integration_tests/suite/test_call_log_generation.py

```diff
@@ -262,6 +263,153 @@ def call_log_is_missing_source_user():
+ BRIDGE_ENTER                   | 2022-07-20 17:56:15.687332        | Harry Potter      | 1603      | 1603      | s         | user       | PJSIP/cul113qn-0000000c  | 1658354172.12  | 1658354172.12  | {"bridge_id":"e2cfd944-9a3e-4885-9ee9-9d1b33886ecf","bridge_technology":""simple_bridge"}
+ BRIDGE_EXIT                    | 2022-07-20 17:56:18.109222        | Willy Wonka       | 1604      | 1604      |           | mycontext  | PJSIP/xfloi61j-0000000d  | 1658354172.12  | 1658354172.12  | {"bridge_id":"e2cfd944-9a3e-4885-9ee9-9d1b33886ecf","bridge_technology":"simple_bridge"}
+ HANGUP                         | 2022-07-20 17:56:18.11027         | Willy Wonka       | 1604      | 1604      |           | mycontext  | PJSIP/xfloi61j-0000000d  | 1658354172.12  | 1658354172.12  | {"hangupcause":16,"hangupsource":"PJSIP/xfloi61j-0000000d","dialstatus":""}
+ CHAN_END                       | 2022-07-20 17:56:18.11027         | Willy Wonka       | 1604      | 1604      |           | mycontext  | PJSIP/xfloi61j-0000000d  | 1658354172.12  | 1658354172.12  |
+ BRIDGE_EXIT                    | 2022-07-20 17:56:18.114121        | Harry Potter      | 1603      | 1603      | s         | user       | PJSIP/cul113qn-0000000c  | 1658354172.12  | 1658354172.12  | {"bridge_id":"e2cfd944-9a3e-4885-9ee9-9d1b33886ecf","bridge_technology":"simple_bridge"}
+ HANGUP                         | 2022-07-20 17:56:18.116943        | Harry Potter      | 1603      | 1603      | s         | user       | PJSIP/cul113qn-0000000c  | 1658354172.12  | 1658354172.12  | {"hangupcause":16,"hangupsource":"PJSIP/xfloi61j-0000000d","dialstatus":"ANSWER"}
+ CHAN_END                       | 2022-07-20 17:56:18.116943        | Harry Potter      | 1603      | 1603      | s         | user       | PJSIP/cul113qn-0000000c  | 1658354172.12  | 1658354172.12  |
+ LINKEDID_END                   | 2022-07-20 17:56:18.116943        | Harry Potter      | 1603      | 1603      | s         | user       | PJSIP/cul113qn-0000000c  | 1658354172.12  | 1658354172.12  |
+'''
+    )
+    def test_internal_call_has_destination_details_setup_correctly(self):
+        user_uuid = 'c3f297bd-93e1-46f6-a309-79b320acb7fb'
+        user_name = 'Willy Wonka'
+        self._assert_last_call_log_matches(
+            '1658354172.12',
+            has_properties(
+                direction='internal',
+                destination_details=contains_inanyorder(
+                    has_properties(
+                        destination_details_key='user',
+                        destination_details_value=user_uuid,
+                    ),
+                    has_properties(
+                        destination_details_key='user',
+                        destination_details_value=user_name,
+                    ),
```

Comment:
This should expect:
```
destination_details_key = 'type', value = 'user'
destination_details_key = 'user_uuid', value = user_uuid
destination_details_key = 'user_name', value = user_name
```
The same remark goes for the other types.

Reply:
I have added the [example entries table](https://github.com/wazo-platform/wazo-call-logd/pull/162#pullrequestreview-1067360863) in the [ticket specs](https://wazo-dev.atlassian.net/browse/WAZO-2826)

Reply:
you mean something like this:
```python
 destination_details=contains_inanyorder(
                    has_properties(
                        destination_details_key='type',
                        destination_details_value='user',
                    ),
                    has_properties(
                        destination_details_key='user_uuid',
                        destination_details_value=user_uuid,
                    ),
                   has_properties(
                        destination_details_key='user_name',
                        destination_details_value=user_name,
                    ),
```

Reply:
Yes, that's right, in order to follow the [specs](https://wazo-dev.atlassian.net/browse/WAZO-2826) described in the ticket.